### PR TITLE
Expose canReadWithCamera as a constant

### PR DESCRIPTION
**Pull request by sbycrosz**
https://github.com/Kerumen/react-native-awesome-card-io/issues/6 🎉

### CODE REVIEW

File: android/src/main/java/com/cardio/RNCardIOUtilities.java [hunk 1]
```diff
@@ -18,8 +19,10 @@ public String getName() {
     return "RCTCardIOUtilities";
   }
 
-  @ReactMethod
-  public void canReadCardWithCamera(Callback callback) {
-    callback.invoke(CardIOActivity.canReadCardWithCamera());
+  @Override
+  public Map<String, Object> getConstants() {
+    final Map<String, Object> constants = new HashMap<>();
+    constants.put("CAN_READ_CARD_WITH_CAMERA", Boolean.valueOf(CardIOActivity.canReadCardWithCamera()));
```
Comment (Kerumen) [inline android/src/main/java/com/cardio/RNCardIOUtilities.java]: The boxing is unnecessary here (`canReadCardWithCamera` already returns a `boolean`), can you remove it?

Comment (sbycrosz) [inline android/src/main/java/com/cardio/RNCardIOUtilities.java]: Thanks! I forgot that Java has **autoboxing** unlike objective-c

**Comment by sbycrosz** [comment]
I updated the PR! 🎉

**Comment by Kerumen** [comment]
Thanks 👍

**Pull request merged**